src/doubleTrouble.js

```diff
@@ -26,7 +26,15 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(array, target) {
-  // write code here
+  const filteredArray = array.slice();
```

Comment:
It's cool that you are copying an array - I mess it in other reviews.
But, will be better to use spread operator
```suggestion
  const filteredArray = [...array];
```
Also, it is not filtered maybe `copiedArray`